modules/blocks/utils.js

```diff
@@ -133,9 +133,9 @@ Utils.prototype.readDbRows = function(rows) {
  */
 Utils.prototype.loadBlocksPart = function(filter, cb) {
 	self.loadBlocksData(filter, (err, rows) => {
-		var blocks = [];
+		var blocks;
```

Comment:
`loadBlocksPart` is called here:
https://github.com/LiskHQ/lisk/blob/4b486acad905ac7c930021f31cd167805986b87d/modules/blocks/chain.js#L547-L555
It's expecting `Array` type there, if `undefined` passed and no error (idk if that can happen, but check there assumes such a possibility) it will result with `TypeError: Cannot read property 'length' of undefined`. So we should ensure consistency there.

Reply:
This change was done to make it consistent with other function return failures within module, where undefined is returned on error.